finance-api/user-guides/fetch-purchase-information-for-card-transactions.md

```diff
@@ -0,0 +1,181 @@
+Fetch purchase information for card transactions
+===
+Using the Finance API and the Purchase API, you can fetch purchase information for transactions of payments and refunds that are made with a Zettle card reader. 
+
+* [Prerequisites](#prerequisites)
+* [Step 1: Find the card transaction UUID](#step-1-find-the-card-transaction-uuid)
+* [Step 2: Fetch purchase information for card transactions](#step-2-fetch-purchase-information-for-card-transactions)
+* [Related task](#related-task)
+* [Related API reference](#related-api-reference)
+
+## Prerequisites
+* Make sure that authorization is set up using [Authorization OAuth2 API](../../authorization.adoc). 
+<!-- to be continued if any -->
+
+## Step 1: Find the card transaction UUID
+Find the transaction UUID for card transactions for which you will fetch purchase information.  
+
+1. Optional: Fetch your organisation UUID. 
+   > **Tip:** You don't need to fetch the organisation UUID, as you can specify the organisation UUID as `self` in requests that require it.
+
+    ```
+    GET users/me
+    ```
+   Example:
+       
+   The following example response returns organisation UUID `ab305d54-75b4-431b-adb2-eb6b9e546013`.
+
+    ```
+    {
+        "uuid": "de305d54-75b4-431b-adb2-eb6b9e546014",
+        "organizationUuid": "ab305d54-75b4-431b-adb2-eb6b9e546013"
+    }
+    ```
+       
+2. Fetch the account card transactions. 
+   > **Tip:** You specify the organisation UUID as `self`.
+
+    ```
+    GET /organizations/{organizationUuid}/accounts/{accountTypeGroup}/transactions?{start}&{end}&includeTransactionType=CARD_PAYMENT&includeTransactionType=CARD_REFUND
+    ```
+   Example:
+   
+   The following example request fetches card transactions of the merchant's Zettle preliminary account.
+   
+   ```
+   GET /organizations/self/accounts/preliminary/transactions?start=2017-01-01&end=2021-06-07&includeTransactionType=CARD_PAYMENT&includeTransactionType=CARD_REFUND
+   ```
+       
+   The following example response returns `originatingTransactionUuid` as `a4b1fc70-3427-11e9-89d7-88df8a0a0855` for a card payment transaction and `originatingTransactionUuid` as `d8254410-4f3f-11eb-a57f-1aef6b55e380` for a card refund transaction.
```

Comment:
according to the example is the opposite 😄  `a4b1fc70-3427-11e9-89d7-88df8a0a0855` for refund and `d8254410-4f3f-11eb-a57f-1aef6b55e380`for payment

Reply:
Fixed it and some formatting issues now.